src/domain/Phone/WebRTCPhone.js

```
@@ -298,20 +298,25 @@ export default class WebRTCPhone extends Emitter implements Phone {
     if (!navigator.mediaDevices) {
       return null;
     }
-    const screenShareStream = await navigator.mediaDevices.getDisplayMedia({
-      video: {
-        cursor: 'always',
-      },
-      audio: false,
-    });
 
+    return navigator.mediaDevices.getDisplayMedia({ video: { cursor: 'always' }, audio: false })
+      .then(stream => {
+        const screenShareStream = stream;
+        // $FlowFixMe
+        screenShareStream.local = true;
+        return this._continueScreenSharing(stream, constraints);
+      })
+      .catch(e => {
+        console.warn(e);
+        return null;
```

Can you return the error so we can catch it ?